roles/django_app_k8s/README.md

```diff
@@ -19,6 +19,8 @@ Requirements
 * Access to the K8s cluster and the relevant namespace
 * `kubectl` commandline tool should be installed on your local machine (Ansible control
   node) to validate/debug deployments
+* `helm` commandline tool should be installed on your local machine (Ansible control
+  node) to deploy some components
```

Comment:
Shouldn't we also specify the `kubernetes` collection in our own dependencies? https://github.com/maykinmedia/commonground-ansible/blob/main/galaxy.yml#L45